livekit_egress.proto

```diff
@@ -258,6 +286,12 @@ message UpdateStreamRequest {
   repeated string remove_output_urls = 3;
 }
 
+message UpdateOutputsRequest {
```

Comment:
wdyt?
```suggestion
message UpdateImageOutputsRequest {
```

Reply:
My initial logic was that this message could be eventually used to update other kinds of output. If we don't force this happening, sure.

Reply:
ah understood.. yeah if we are going to use it for other outputs.. then what we have is better